CHANGELOG.md

```diff
@@ -9,33 +9,35 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 **ENHANCEMENTS**
 - Add support for AWS Trainium instances.
 - Add support for Slurm Accounting.
-- Upgrade third-party cookbook dependencies:
-  - selinux-6.0.5 (from selinux-6.0.4)
-  - nfs-5.0.0 (from nfs-2.6.4)
-- Upgrade EFA installer to `1.18.0`
-  - Efa-driver: `efa-1.16.0-1`
-  - Efa-config: `efa-config-1.11-1`
-  - Efa-profile: `efa-profile-1.5-1`
-  - Libfabric-aws: `libfabric-aws-1.16.0~amzn4.0-1`
-  - Rdma-core: `rdma-core-41.0-2`
-  - Open MPI: `openmpi40-aws-4.1.4-2`
-- Upgrade NICE DCV to version `2022.1-13300`.
-  - server: `2022.1.13300-1`
-  - xdcv: `2022.1.433-1`
-  - gl: `2022.1.973-1`
-  - web_viewer: `2022.1.13300-1`
+- Add support for compute resources with multiple instance types.
+- Add support for mounting and unmounting shared storage at cluster update.
 - Configure NFS threads to be `min(256, max(8, num_cores * 4))` to ensure better stability and performance.
+- Move NFS installation at build time to reduce configuration time.
```

Comment:
I'd move into CHANGES, as:
- Install NFS at image build time.

Reply:
That change is providing a performance improvement. I think it's worth it to mention in enhancement.
Why do you think it's not?